osu.Game/Skinning/LegacySkinExtensions.cs

```diff
@@ -63,8 +64,51 @@ IEnumerable<Texture> getTextures()
             }
         }
 
-        public static bool HasFont(this ISkin source, string fontPrefix)
-            => source.GetTexture($"{fontPrefix}-0") != null;
+        public static bool HasFont(this ISkin source, LegacyFont font)
+        {
+            return source.GetTexture($"{source.GetFontPrefix(font)}-0") != null;
+        }
+
+        public static string GetFontPrefix(this ISkin source, LegacyFont font)
```

Comment:
Weakly disagree against making these public. A major part of this exercise is to unify the handling of texts to be consistent. Exposing these as public should only be done if a usage that requires it pops up.

Would move to `LegacySpriteText` for now.

Reply:
Hmm, quite a fair point.

Reply:
Ah, actually..:

<img width="713" alt="image" src="https://user-images.githubusercontent.com/22781491/112532264-af08fc80-8db9-11eb-86a5-0eecaa1597c6.png">

Reply:
Could make `HasFont` a static method of `LegacyFont`, I suppose? Maybe not worth it. Not sure.